### DISCUSSION TITLE
​Enhanced backlinks

### DISCUSSION THREAD

**Discussion by MingStudentSE** [toeverything/AFFiNE#7809] [Ideas]
### Description

Now we just can know the page from backlinks.
I want to know the content and time.
![image](https://github.com/user-attachments/assets/afcafcfc-d244-473e-92b9-5374993edd0b)
![image](https://github.com/user-attachments/assets/7ff1efe1-4551-41c9-9392-fce39cf55d93)
Could you please tell me is it achievedable?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

[a comment by an automated account is not shown]

**Comment by forehalo** [comment]
@hwangdev97

**Comment by hwangdev97** [comment]
Yes, we have also realized that our backlinks are not good enough, and we will continue to spend time improving them.
